projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -47,6 +49,8 @@ describe('AuthService', () => {
     locationSpy = jasmine.createSpyObj('Location', ['path']);
     locationSpy.path.and.returnValue('');
 
+    authClientConfigSpy = jasmine.createSpyObj(['get']);
```

Comment:
I have a similar question as from the other PR - `AuthClientConfig` is such a simple type that I'm not convinced we should be mocking it. It's not a "seam" in the app, it just gets and sets another object. Do we really need to mock this or can we just give it some config to work with?

Just thinking of the future where we might actually change the internals of `AuthClientConfig` and how it could unexpectedly break things in such a way that we wouldn't know about it, and also generally the less mocking we do the better.

We can make the code neat and tidy, but let's not mock it if possible. And whatever we do here, we should also do on the other PR.

Reply:
I updated the PR to avoid the mocking. Will need to try and fix codecov...